test/system/035-logs.bats

```diff
@@ -30,6 +30,23 @@ load helpers
     run_podman rm $cid
 }
 
+@test "podman logs - tail test" {
+    rand_string=$(random_string 40)
+
+    run_podman run -d $IMAGE sh -c "echo test1; echo test2"
+    cid="$output"
+
+    run_podman logs --tail 1 $cid
+    is "$output" "test2"  "logs should only show last line"
+
+    run_podman restart $cid
+
+    run_podman logs --tail 1 $cid
+    is "$output" "test2"  "logs should only show last line"
```

Comment:
Could you rename this message? Maybe "logs, after restart, should still only show last line"? Distinct error messages make it sooooo much easier to track down the exact failure.